app/views/payments/_form.html.haml

```diff
@@ -0,0 +1,68 @@
+- inv_amt = @payment.invoice.total_amount
+- pay_amt = @payment.invoice.total_amount_payments
```

Comment:
Avoid using abbreviated names for variables.
rename `inv_amt` to `invoice_amount`
rename `pay_amt` to `paid_amount`